src/pages/plugins/index.md.ejs

```diff
@@ -2,17 +2,190 @@
 layout: content.html
 ---
 
+<%
+// Plugin data
+// TODO: put badges into badge object to iterate over
+// Badges are all seneca-gateway temporarily
+let pluginsByGroup = [
```

Comment:
make this a `const` to be more *defensive* - prevents inadvertent mistakes

Reply:
Updated